pkg/constants/constants.go

```diff
@@ -12,6 +12,7 @@ const (
 	KibanaTrustedCAName        = "kibana-trusted-ca-bundle"
 	SecretHashPrefix           = "logging.openshift.io/"
 	KibanaInstanceName         = "kibana"
+	ElasticsearchInstanceName  = "elasticsearch"
```

Comment:
so this is where things get weird... this shouldn't be a constant (nor should "kibana"... ) we get the names based on the CR and should accept any that come to us, its just that CLO will always use the names `kibana` and `elasticsearch`